### PR TITLE
doc(reports): 13743 - add doc for reports

### PR DESCRIPTION
**Pull request by IlyaIzr**

### CODE REVIEW

File: src/features/reports/readme.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+}
+```
+
+### How it works
+
+#### Data model
+
+There are 2 dynamic configs we receive from the back-end.
+
+1. Let's call it `"Reports List"`. It's an array of available reports containing all the info about each one of them. On of it's properties is the link to `.csv` file. All the keys stored in `./atoms/reportsAtom.ts` `Report` type.
```
Comment (vkozio) [inline src/features/reports/readme.md]: Proposal:

`"Reports List"` - array of available reports containing all info about each of them including link to `.csv` file with report data.

File: src/features/reports/readme.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
+## Reports
+
+This feature is responsible for presenting a list of Kontur reports and browsing them.
+Each report has a link, a description, date of latest update and report's content represented as a table. Originally those reports are stored as `.csv` tables
+
+### How to use
+
+There are 2 separate root components for that feature.
+`<ReportsList />` shows the list of reports, their brief description and general description of reports section. It also allows to navigate to the report by clicking on its card.
+`<ReportInfo />` provides whole info about the given report, taking it's id from the link.
```
Comment (dqunbp) [inline src/features/reports/readme.md:10]: It's not clear where the `.csv` data comes from, does it fetches when the component mounts by report id?

File: src/features/reports/readme.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+```
+
+### How it works
+
+#### Data model
+
+There are 2 dynamic configs we receive from the back-end.
+
+1. Let's call it `"Reports List"`. It's an array of available reports containing all the info about each one of them. On of it's properties is the link to `.csv` file. All the keys stored in `./atoms/reportsAtom.ts` `Report` type.
+2. The data received from `.csv` link is the second data model. Let's call it `"Report Content"`.
```
Comment (dqunbp) [inline src/features/reports/readme.md]: It's not clear where and when the `.csv` data comes from, does it fetches when the component mounts by report id?

Comment (IlyaIzr) [reply src/features/reports/readme.md]: It comes from resource `reportResourceAtom` as described below on the `Logic` section. This section is about data models themselves

**Review by Akiyamka — APPROVED** [review]

**Pull request merged**